src/ensembl/src/shared/components/help-popup/types.ts

```diff
@@ -14,31 +14,37 @@
  * limitations under the License.
  */
 
-export type HelpVideo = {
-  title: string;
-  description: string;
+export type ArticleType = 'article' | 'video';
+
+export type Article = {
+  slug: string; // an article identtifier
+  url: string;
+  title: string; // part of article metadata; for use in the page meta tag
+  description: string; // part of article metadata; for use in the page meta tag
+  related_articles: RelatedArticle[];
+};
+
+export type TextArticle = Article & {
+  type: 'article';
```

Comment:
Wouldn't it be better to call this something like `text-article` instead? 
I feel it's slightly confusing to have Article of type `article`.

Reply:
Umm. This is tightly linked to what the help&docs api is returning. Compare the responses:

- [for a text article](http://update-help.review.ensembl.org/api/docs/article?slug=species-selector)
- [for a video article](http://update-help.review.ensembl.org/api/docs/article?slug=ensembl-help-getting-started-videos-find-and-add-species)

The api is ours, and we can of course change what it responds with, but the `type: "video"` in the response just feels so much nicer than something like `type: "video_article"`...

Reply:
As discussed during the code review, we have decided to leave the current implementation as it is.